tests/test_output.py

```diff
@@ -456,6 +456,7 @@ def check_grid_search_cv_results(task, do_grid_search):
     # off, so simply turn it off here as well since it should
     # result in the same situation
     elif task == 'learning_curve':
+        values_to_fill_dict['metrics'] = values_to_fill_dict.pop('objectives')
```

Comment:
Why is this needed?

Reply:
because, `learning_curve` does not support `objectives` but instead will need `metrics`.

Reply:
Yeah but I thought any tasks other than train and predict raise a ValueError anyway (see line 476 below)

Reply:
that is only for `train` and `predict`.

Reply:
I mean, ValueError is raised only if it is `train` and `predict` and `do_grid_search` is False. For the rest tasks, it will run `run_configuration(config_path, quiet=True)` in line 481

Reply:
I see 👍